modules/nested-tabs/assets/scss/frontend.scss

```diff
@@ -229,6 +229,21 @@
 	}
 }
 
+// TODO: Remove once PHP global color support is fixed.
+:is( .elementor .elementor-element.elementor-widget-n-tabs > .elementor-widget-container > .e-n-tabs > .e-n-tabs-heading .e-n-tab-title,
+  .elementor .elementor-element.elementor-widget-n-tabs > .elementor-widget-container > .e-n-tabs > .e-n-tabs-content .e-n-tab-title ){
+
```

Comment:
Can we add the normal styling here as well?

Reply:
We don't need to. 
This solution solves the malfunction of the PHP global colors, in the normal state we are not using global colors.